### PR TITLE
Small musig improvements

### PR DESCRIPTION
**Pull request by jonasnick**

### CODE REVIEW

File: include/secp256k1_musig.h [hunk 1]
```diff
@@ -409,12 +421,12 @@ SECP256K1_API int secp256k1_musig_partial_sign(
  *           verify, 1 otherwise
  *  Args         ctx: pointer to a context object, initialized for verification
  *  In:  partial_sig: pointer to partial signature to verify
```
Comment (robot-dreams) [inline include/secp256k1_musig.h:423]:
```suggestion
 *  In:  partial_sig: pointer to partial signature to verify, sent by
 *                    the signer associated with `pubnonce` and `pubkey`
```

**Review by robot-dreams — COMMENTED** [review]
ACK 1e95f2376ba935424888b11c4f7a72a758854211

Good catch on the extra comment in `session_impl.h`. All inline comments are nits, feel free to ignore

File: examples/musig.c [hunk 2]
```diff
@@ -4,9 +4,9 @@
  * file COPYING or https://www.opensource.org/licenses/mit-license.php.*
  **********************************************************************/
 
-/**
- * This file demonstrates how to use the MuSig module to create a multisignature.
- * Additionally, see the documentation in include/secp256k1_musig.h.
+/** This file demonstrates how to use the MuSig module to create a
+ *  multisignature. Additionally, see the documentation in
```
[the remark this answers is not in the document]
Comment (jonasnick) [reply examples/musig.c]: This sounds a bit like m < n is possible with musig, just not supported. I changed "multisignature" to "3-of-3 multisignature instead.

**Comment by robot-dreams** [comment]
ACK d895b10c18b8aa19a77f0a80f318e1a2052e7c9b

**Review by real-or-random — APPROVED** [review]
ACK d895b10c18b8aa19a77f0a80f318e1a2052e7c9b

**Pull request merged**